src/experiments/kfold_sonar.py

```diff
@@ -0,0 +1,96 @@
+"""
+test with new config
+
+"""
+
+import os
+import random
+
+import numpy as np
```

Comment:
This whole file looks like a copy of kfold_opportunity with line 25 changed. Why do we have both files?